includes/class-wc-stripe-logger.php

```diff
@@ -244,6 +244,10 @@ public static function debug( $message, $context = [] ) {
 	 * @return boolean
 	 */
 	public static function can_log(): bool {
+		if ( WC_Stripe_Helper::is_enhanced_debug_mode_enabled() ) {
```

Comment:
Hmm... I now see that we're adding a general enhanced logging flag rather than something specific to OAuth. Might it be worth naming this something related to verbose logging rather than enhanced logging?